core/Symbols.cc

```diff
@@ -1472,6 +1472,7 @@ void Symbol::recordSealedSubclass(MutableContext ctx, ClassOrModuleRef subclass)
     }
 }
 
+// Returns the locations that are allowed to subclass the sealed class.
```

Comment:
Not sure how much this matters to you, but clangd will show documentation like this on hover if it's in the header instead of the implementation.

Reply:
Good point, I'll move it over there.